agent/service.go

```diff
@@ -155,7 +174,7 @@ func proxyIO(ctx context.Context, path string, port uint32, in bool) {
 	}
 }
 
-func unpackBundle(path string, bundle *types.Any) (*types.Any, error) {
+func unpackBundle(path string, bundle *types.Any) (*proto.ExtraData, error) {
```

Comment:
Why the change from types.Any to proto.ExtraData?

Reply:
To get access to the drive id on the extra data